au3/libraries/lib-track/Track.h

```diff
@@ -92,8 +92,11 @@ class TrackId
    bool operator <  (const TrackId &other) const
    { return mValue <  other.mValue; }
 
+   inline operator int64_t() const { return mValue; }
+   inline int64_t raw() const { return mValue; }
+
 private:
-   long mValue;
+   int64_t mValue = -1;
```

Comment:
Stop using -1 as a “magic” value to represent an uninitialized state. This can be confusing and easily misinterpreted as a valid ID. Instead, use std::optional<int64_t>, which explicitly shows whether the ID is set.

Reply:
Tell us more here, usually they don't use optinal for ID because they need performance, very often they search by ID, for example... What is your experience using optinal in such cases?